bin/aws/ssm_nomad_server.sh

```diff
@@ -1,13 +1,38 @@
 #!/usr/bin/env bash
```

Comment:
Since all these scripts are basically the same apart from the tag used in the instance filter and the port that needs to be forwarded, it might be good to put all that into a common library that is sourced by the individual scripts, which then just parameterize the calls.

Reply:
Done